edx-platform/pearson-bsg-theme/lms/templates/certificates/_accomplishment-header.html

```diff
@@ -0,0 +1,6 @@
+<!-- Style attached in this file because it is outside lms/static/sass/ and it in not being applied when overrided -->
```

Comment:
```suggestion
<!-- Style attached in this file because it is outside lms/static/sass/ and it is not being applied when overridden -->
```
Please apply this grammar suggestion to all files.

The Kaust and LBS themes are missing from this PR, any particular reason not to apply these changes to those themes? @ivanvgh

Reply:
I discard those since they're are not working in production, specifically for certificates. I wanted to ask if those sites which are using those themes are available if so we can open a new ticket to check what is going on with those themes int particular.

Reply:
Ok, please move this discussion forward to see what we have to do with those themes. @ivanvgh

Reply:
O.k. I will let Scott know.